text-to-sql-app/src/app/globals.css

```
@@ -0,0 +1,222 @@
+/* Reset default styles from Next.js */
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Sans

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)